src/ebonite/core/objects/wrapper.py

```diff
@@ -237,6 +237,13 @@ def resolve_method(self, method_name=None):
             self._check_method(method_name)
         return method_name
 
+    def match_methods_by_type(self, input: DatasetType, output: DatasetType) -> typing.List[str]:
```

Comment:
How it will work when there are multiple matching methods? We will just evaluate metric for all of them?

Reply:
yes. at the end of model.evaluate you can see that we check if we have only one method - we return its result, but if there are multiple - we return dict